README.md

```diff
@@ -25,6 +25,27 @@ Based on traditional API Gateways, it intercepts all incoming messages moving th
 
 It supports all the protocols AsyncAPI supports through [bindings](https://github.com/asyncapi/bindings).
 
-This **Event Gateway** is also compatible with the HTTP protocol, natively or trough an external provider like [Krakend.io](http://krakend.io).
+This **Event Gateway** is also compatible with the HTTP protocol, natively or through an external provider like [Krakend.io](http://krakend.io).
 
-AsyncAPI also offers a Marketplace where users can install middlewares for the **Event Gateway** made by the community and also some officially maintained by the AsyncAPI org.
+AsyncAPI also offers a Marketplace where users can install middlewares for the **Event Gateway** made by the community or officially maintained by the AsyncAPI org.
+
+# Goals
+
+## 1. Performance-first while ensuring message delivery
+The **Event Gateway** it's a **stateless** solution that ensures messages are delivered as fast as possible using a minimal resource footprint.
+Delivering messages as a top priority means no data loss should happen.
+
+## 2. Transparent usage.
+No change in the user's code is needed. The service reads and responds pretending to be the final broker. 
+Messages infer the protocol based on the shape of the input network packet.
```

Comment:
This is going to be interesting 😄

Reply:
Interesting and challenging, especially considering the first bullet.
However, remember those are just the goals based on a vision of how I see this service should be in the future (I would say years).
We can always modify them in case we see any of those are not of our interest anymore (For example, technically not worth the effort).